gci/logo.py

```diff
@@ -0,0 +1,27 @@
+import os
+import requests
+
+
+def get_logo():

```

Comment:
This function belongs in the GitOrg class of `gitorg.py`.  There it already has the org name.  The function needs an optional 'size' , and returns a blob .

Let the caller (`view.py`) decide how that blob should be used.

Reply:
How would I go about accessing the org name from the functions contained within `gitorg.py`?